platform/ui/src/components/SidePanel/SidePanel.jsx

```diff
@@ -65,10 +65,11 @@ const SidePanel = ({
             variant="text"
             color="inherit"
             rounded="none"
+            style={{ height: '52px' }}
```

Comment:
This is very specific? There are many height classes that maybe could be used, as `height` extends from `spacing`:

https://github.com/OHIF/Viewers/blob/92805dd9b6f31f51cd58705597ea711b2f54e9d5/platform/ui/tailwind.config.js#L152-L174